IM_calculation/scripts/calculate_ims.py

```diff
@@ -136,6 +136,13 @@ def load_args():
         type=int,
         help="Please provide the number of processors. Default is 1",
     )
+
+    parser.add_argument(
+        "--real_stats_only",
+        action="store_true",
+        help="Please add '-r' to indicate if you want to consider real stations only",
```

Comment:
An explicit '-r' would be nice, but should be covered by argparse automatically adding abbreviations.

Reply:
Conflicts with -r --rupture

Reply:
Then the comment is wrong as adding -r is for --rupture, and will cause a crash if someone does try -r for --real_stats_only. Can you change the comment and matching documentation?

Reply:
Thanks for catching this!